initializr-generator/src/main/java/io/spring/initializr/generator/buildsystem/maven/MavenBuild.java

```diff
@@ -134,4 +138,66 @@ public String getPackaging() {
 		return this.packaging;
 	}
 
+	public void resource(String directory, Consumer<ResourceBuilder> customizer) {
+		customizer.accept(this.resources.computeIfAbsent(directory, (key) -> new ResourceBuilder(directory)));
+	}
+
+	public List<Resource> getResources() {
+		return this.resources.values().stream().map(ResourceBuilder::build).collect(Collectors.toList());
+	}
+
+	/**
+	 *
+	 * Builder to create a {@link Resource}.
+	 *
+	 *
+	 */
+	public static final class ResourceBuilder {
+
+		private String directory;
+
+		private List<String> includes = new ArrayList<>();
+
+		public ResourceBuilder(String directory) {
+			this.directory = directory;
+		}
+
+		Resource build() {
+			return new Resource(this.directory, this.includes);
+		}
+
+		public ResourceBuilder include(String include) {
```

Comment:
Looks like a missed opportunity to be able to specify multiple include patterns at once.